libraries/chain/include/eosio/chain/fork_database.hpp

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include <fc/io/cfile.hpp>
 #include <eosio/chain/block_state_legacy.hpp>
 #include <eosio/chain/block_state.hpp>
 #include <eosio/chain/block_handle.hpp>
```

Comment:
I thought I had removed this include and the `log_fork_comparison`. I think it would be cleaner for `fork_database` to not know about the internals of `block_handle`. Seems we should only work with `block_state` and `block_state_legacy` in fork database.

Reply:
I removed the `block_handle.hpp` include.